manifests/f/FACEITLTD/FACEITClient/1.31.12/FACEITLTD.FACEITClient.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2023/Feb/16
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: FACEITLTD.FACEITClient
+PackageVersion: 1.31.12
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://faceit-client.faceit-cdn.net/release/FACEIT-setup-latest.exe
+  InstallerSha256: 324ECA9CE913C5C72D39E7A35857ED34BD6BCDE11B6C00DAFE096EBE8B3FD6E8

```

Comment:
```suggestion
  InstallerSha256: B1FF742E204033DA41F7948C5AA291714CA959689221D6540E901C6FF428EBB9
```

Reply:
I'm seeing this same hash mismatch:
```
PS C:\ManVal> Get-FileHash .\FACEIT-setup-latest.exe

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          B1FF742E204033DA41F7948C5AA291714CA959689221D6540E901C6FF428EBB9       C:\ManVa...
```

Reply:
It seems to be a CDN issue:
![image](https://user-images.githubusercontent.com/11600822/219766608-d9cdd10d-b63b-4400-a4b7-972cfcfbd9cf.png)

![image](https://user-images.githubusercontent.com/11600822/219766793-04a0f2bf-b4df-405b-b153-a7f4a2de2577.png)

Reply:
Yeah, I've just tried it again and now I'm seeing `324eca9ce913c5c72d39e7a35857ed34bd6bcde11b6c00dafe096ebe8b3fd6e8` as well.

Reply:
The hash appears to have reverted:

```
PS C:\ManVal> Get-FileHash .\FACEIT-setup-latest.exe

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          324ECA9CE913C5C72D39E7A35857ED34BD6BCDE11B6C00DAFE096EBE8B3FD6E8       C:\ManVal\Installers\FACEIT-...
```